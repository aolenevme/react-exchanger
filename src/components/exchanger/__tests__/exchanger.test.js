import React from "react";
import renderer from "react-test-renderer";

import Exchanger from "../exchanger.js";

describe("<Exchanger />", () => {
    it("is rendered correctly", () => {
        expect(renderer
            .create(<Exchanger/>)
            .toJSON()).toMatchSnapshot();
    });
});
