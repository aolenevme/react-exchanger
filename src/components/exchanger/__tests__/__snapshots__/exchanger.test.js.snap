// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Exchanger /> is rendered correctly 1`] = `
.c2 {
  font-size: 3rem;
  line-height: 3rem;
  color: #f3f6ff;
}

.c3 {
  font-size: 3rem;
  line-height: 3rem;
  color: #f3f6ff;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  padding: 0;
  width: 100%;
  height: 3rem;
  border: 0 solid transparent;
  border-right: 1px solid transparent;
  background-color: transparent;
  cursor: pointer;
}

.c3:focus-within {
  -webkit-animation: caret-pulse 1.5s cubic-bezier(.215,.61,.355,1) forwards infinite;
  animation: caret-pulse 1.5s cubic-bezier(.215,.61,.355,1) forwards infinite;
}

.c4 {
  margin: 0;
  padding: 0;
  width: 0;
  height: 0;
  border: 0;
  -moz-appearance: textfield;
}

.c4::-webkit-outer-spin-button,
.c4::-webkit-inner-spin-button {
  -webkit-appearance: none;
}

.c4:invalid {
  box-shadow: none;
}

.c6 {
  font-size: 0.8rem;
  color: #7199e6;
}

.c0 {
  width: 100%;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 1rem;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0 0.3rem;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <span
      className="c2"
    >
      USD
    </span>
    <label
      className="c3"
    >
      -
      145.57
      <input
        className="c4"
        disabled={false}
        onInput={[Function]}
        type="number"
        value={145.57}
      />
    </label>
  </div>
  <div
    className="c5"
  >
    <span
      className="c6"
    >
      You have 58.33 dollars
    </span>
    <span
      className="c6"
    >
      Â£1 = $1.45
    </span>
  </div>
</div>
`;
